examples/hello-world/src/test/scala/com/jakehschwartz/finatra/swagger/SampleAppBaseTest.scala

```diff
@@ -0,0 +1,23 @@
+package com.jakehschwartz.finatra.swagger
+
+import com.google.inject.Stage
+import com.twitter.finatra.http.EmbeddedHttpServer
+import com.twitter.inject.server.FeatureTest
+import org.scalatest.Matchers
+
+trait SampleAppBaseTest extends FeatureTest with Matchers {
+
+  override protected def beforeEach(): Unit = {
+    server.assertStarted()
+    server.assertHealthy()
+  }
+
+  def stage: Stage = Stage.DEVELOPMENT
+  override def server: EmbeddedHttpServer
+
+  protected def makeServer(serverName: String): EmbeddedHttpServer = new EmbeddedHttpServer(
```

Comment:
an EmbeddedHttpServer shouldn't be shared across test classes.

Reply:
It wouldn't be shared if it was a val, each instance of a SampleAppBaseTest would have its own

Reply:
see https://twitter.github.io/finatra/user-guide/testing/feature_tests.html#sharing-a-server-fixture-between-many-feature-tests

Reply:
Makes sense. Could you put a comment in the code so no one accidentally changes it in the future? After that, Ill merge this in a get a patch build going.

Reply:
done, thanks!